domains/authorization/Authorization.API/API.UnitTests/API.UnitTests/Mappers/ClientTypeMapperTest.cs

```diff
@@ -0,0 +1,86 @@
+using API.Authorization.Controllers;
+using FluentAssertions;
+
+namespace API.UnitTests.Mappers;
+
+public class ClientTypeMapperTest
```

Comment:
Could this test be handled generically? with something along this for one way:

```
[Fact]
    public void Bla()
    {
        var values = Enum.GetValuesAsUnderlyingType<ApiClientType>();
        foreach (var clientTypeEnumValue in values)
        {
            var apiEnumValue = (ApiClientType)clientTypeEnumValue;
            var persistenceEnumValue = MapToPersistenceModel(apiEnumValue);

            Assert.Equal(Enum.GetName(apiEnumValue ), Enum.GetName(persistenceEnumValue ));
        }
    }
```

Reply:
It could. But it's already typed and not like we are going to have other client types in the future?